### PR TITLE
Add a unit test to the BigQuery sample.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: bigquery/api/GettingStarted/test/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("test")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
```
Comment (remi) [inline bigquery/api/GettingStarted/test/Properties/AssemblyInfo.cs]: Add license header and populate AssemblyCompany and AssemblyCopyright ([here's what we're using in google-api-dotnet-client](https://github.com/google/google-api-dotnet-client/blob/master/Src/GoogleApis.Core/Properties/AssemblyInfo.cs#L26-L28)).

File: bigquery/api/GettingStarted/test/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    /// 
+    /// How to run:
+    /// 1. Set the environment variables:
+    ///    GOOGLE_PROJECT_ID = your project id displayed on the Google Developers Console.
+    ///    GOOGLE_APPLICATION_CREDENTIALS = path to the .json file you downloaded from the
+    ///      Google Developers Console.
+    /// 2. MSTest /testcontainer:test.dll   
+    /// </summary>
+    [TestClass]
+    public class UnitTest1
```
Comment (remi) [inline bigquery/api/GettingStarted/test/UnitTest1.cs]: Rename autogenerated file/class name to something meaningful.

`Tests.cs` would be fine or `BigQueryTests` - just not the autogenerated name :smile:

**Comment by SurferJeffAtGoogle** [comment]
Ready for another look.

**Comment by remi** [comment]
:+1:

**Pull request merged**